dotnet/README.md

```diff
@@ -0,0 +1,51 @@
+## Getting Started with SFTP using C#(.NET)
+
+#Project Description
+
+The project contains two solutions:
+- SFTPClient: This is a simple console application which depicts how we can use our SFTP service to perform differnet file operations.
+- SFPTService: This is a simple class library project that conatains the contains code to connect, upload , dowload and delete files.
```

Comment:
@paras-lalchandani Fix typos:

`This is a simple class library project that contains code to connect, upload ,download and delete files.

Reply:
@motymichaely done